client/src/App.tsx

```diff
@@ -4,7 +4,7 @@ function App() {
   return (
     <>
       <header>
-        <h1 className="logo">JS Monorepo kebab branch</h1>
+        <h1 className="logo">JS Monorepo kebab branch pour rémi </h1>
```

Comment:
ton h1 doit être écrit en anglais sur cette partie -> "pour rémi" doit être changé en "for Rémi"